apps/ledger-live-desktop/src/renderer/families/bitcoin/SendRecipientFields.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { PureComponent } from "react";
+import { connect } from "react-redux";
+import { isConfirmedOperation } from "@ledgerhq/live-common/operation";
+import Alert from "~/renderer/components/Alert";
+import TranslatedError from "~/renderer/components/TranslatedError";
+import { confirmationsNbForCurrencySelector } from "../../reducers/settings";
+import { withTranslation } from "react-i18next";
+import { getMainAccount } from "@ledgerhq/live-common/account/index";
+import { PendingOperation } from "@ledgerhq/errors";
+
+class SendRecipientFields extends PureComponent {
+  render() {
+    const { confirmationsNb, account } = this.props;
+    const pendingOperationError = new PendingOperation();
+    const operations = account.pendingOperations.concat(account.operations);
+    const incomingTransactionPending =
+      operations.find(
```

Comment:
use "some" method instead of "find" would make the code clearer

Reply:
Done, thanks !